deploy/packages/bbp-packages.yaml

```diff
@@ -125,7 +125,7 @@ packages:
       - nest@2.18.0
       - py-efel@3.0.80
       - py-bluepyefe@0.3.13
-      - py-bglibpy@4.4.18 ^neuron%intel@19.0.4
+      - py-bglibpy@4.4.27 ^neuron%intel@19.0.4
```

Comment:
These config files have been moved to `environments`, and should not require version numbers any longer. Could you please rebase on current `develop`?

Reply:
Ah thanks! This will make further updates easier.